### PR TITLE
Feature/sign in

### PR DESCRIPTION
**Pull request by calum-gray**

### CODE REVIEW

File: src/Interfaces/ActivityLog.php
Comment (AnthonyEdmonds) [inline src/Interfaces/ActivityLog.php:1]: Why is this needed for removing the sign in stuff?

File: tests/Models/User.php
Comment (AnthonyEdmonds) [inline tests/Models/User.php:1]: Why is this needed for removing the sign in stuff?

Comment (calum-gray) [reply tests/Models/User.php:1]: i thought it would be useful to put in now since it replaces the permissions

Comment (AnthonyEdmonds) [reply tests/Models/User.php:1]: Does it work without them? If so, remove them from this branch.

Add things when you actually need them.

**Review by AnthonyEdmonds — APPROVED** [review]
